edx_proctoring/backends/rest.py

```diff
@@ -76,7 +77,22 @@ def get_javascript(self):
         Returns the url of the javascript bundle into which the provider's JS will be loaded
         """
         package = self.__class__.__module__.split('.')[0]
-        bundle_chunks = get_files(package, config="WORKERS")
+        bundle_chunks = ''
+        try:
+            bundle_chunks = get_files(package, config="WORKERS")
+        except WebpackBundleLookupError:
+            log.warning(
+                u'Could not find webpack bundle for proctoring backend %s.' +
```

Comment:
+ isn't required for string continuations. Also, perhaps `warnings.warn(message)` would be better than `log.warning(message)`, because it'll only appear once in the logs.

Reply:
warnings.warn messages are probably excessively noisy as output from running unit tests. They may cause them to fail, as far as CI is concerned. Let's find out!